@bobfang1992 
I see upstream CI is testing windows, but here we're running into compiler errors, unfortunately:
```
  "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -ID:\bld\pytomlpp_1641955540918\work/include -ID:\bld\pytomlpp_1641955540918\work/third_party -ID:\bld\pytomlpp_1641955540918\_h_env\lib\site-packages\pybind11\include -ID:\bld\pytomlpp_1641955540918\_h_env\include -ID:\bld\pytomlpp_1641955540918\_h_env\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.16.27023\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.16.27023\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22000.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22000.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22000.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22000.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22000.0\cppwinrt" -ID:\bld\pytomlpp_1641955540918\_h_env\Library\include -ID:\bld\pytomlpp_1641955540918\_h_env\Library\include /d1trimfile:D:\bld\pytomlpp_1641955540918\work /EHsc /Tpsrc/encoding_decoding.cpp /Fobuild\temp.win-amd64-3.8\Release\src/encoding_decoding.obj /std:c++17
  encoding_decoding.cpp
  src/encoding_decoding.cpp(84): error C2280: 'toml::v3::table::table(void) noexcept': attempting to reference a deleted function
  d:\bld\pytomlpp_1641955540918\_h_env\library\include\toml++\impl/table.h(253): note: see declaration of 'toml::v3::table::table'
  d:\bld\pytomlpp_1641955540918\_h_env\library\include\toml++\impl/table.h(253): note: 'toml::v3::table::table(void) noexcept': function was implicitly deleted because the explicit exception specification is incompatible with that of the implicit declaration
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX64\\x64\\cl.exe' failed with exit code 2
  ERROR: Failed building wheel for pytomlpp
```

I'm going to try if things pass with a newer toolchain...

--- a/.ci_support/win_64_python3.10.____cpython.yaml
+++ b/.ci_support/win_64_python3.10.____cpython.yaml
@@ -3,7 +3,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 cxx_compiler:
-- vs2017
+- vs2019
 pin_run_as_build:
   python:
     min_pin: x.x

--- a/.ci_support/win_64_python3.7.____73_pypy.yaml
+++ b/.ci_support/win_64_python3.7.____73_pypy.yaml
@@ -3,7 +3,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 cxx_compiler:
-- vs2017
+- vs2019
 pin_run_as_build:
   python:
     min_pin: x.x

--- a/.ci_support/win_64_python3.7.____cpython.yaml
+++ b/.ci_support/win_64_python3.7.____cpython.yaml
@@ -3,7 +3,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 cxx_compiler:
-- vs2017
+- vs2019
 pin_run_as_build:
   python:
     min_pin: x.x

--- a/.ci_support/win_64_python3.8.____cpython.yaml
+++ b/.ci_support/win_64_python3.8.____cpython.yaml
@@ -3,7 +3,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 cxx_compiler:
-- vs2017
+- vs2019
 pin_run_as_build:
   python:
     min_pin: x.x

--- a/.ci_support/win_64_python3.9.____cpython.yaml
+++ b/.ci_support/win_64_python3.9.____cpython.yaml
@@ -3,7 +3,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 cxx_compiler:
-- vs2017
+- vs2019
 pin_run_as_build:
   python:
     min_pin: x.x

--- a/recipe/conda_build_config.yaml
+++ b/recipe/conda_build_config.yaml
@@ -1,0 +1,4 @@
+c_compiler:    # [win]
+- vs2019       # [win]
+cxx_compiler:  # [win]
+- vs2019       # [win]

